lib/classes/Swift/Transport/StreamBuffer.php

```diff
@@ -95,7 +95,9 @@ public function startTLS()
         // To support modern tls we allow explicit tls1.0, tls1.1, tls1.2
         // Ssl3 and older are not allowed because they are vulnerable
         // @TODO make tls arguments configurable
-        return stream_socket_enable_crypto($this->_stream, true, STREAM_CRYPTO_METHOD_TLSv1_0_CLIENT | STREAM_CRYPTO_METHOD_TLSv1_1_CLIENT | STREAM_CRYPTO_METHOD_TLSv1_2_CLIENT);
+	// To Support more php versions
+	$supportTls = PHP_VERSION > '5.6' ? STREAM_CRYPTO_METHOD_TLSv1_0_CLIENT | STREAM_CRYPTO_METHOD_TLSv1_1_CLIENT | STREAM_CRYPTO_METHOD_TLSv1_2_CLIENT : STREAM_CRYPTO_METHOD_TLS_CLIENT;
```

Comment:
Shouldn't you rather use `PHP_VERSION_ID` instead?

Reply:
I am careless, thank you for pointing out my mistake. @xabbuh 
Thank you for maintaining this project. @fabpot

Reply:
Ah, I missed that this was already fixed by d9aed422e20f2191d5d4324fce102b3db9b5de6b.